README.md

```diff
@@ -28,102 +28,37 @@ the mailing list and post your new questions there.
 
 ## How to Build GNU Radio
```

Comment:
I'd recommend changing this header to "How to Obtain GNU Radio" or "How to Install GNU Radio" ... "build" to me implies compiling, which the immediate next paragraph is not about. I like the new paragraph, so let's fix this header.

Reply:
"How to Install GNU Radio" sounds good. There are actually three sub-topics here, so to be consistent, I suggest putting "Package Distributions" as a sub-header right under "How to Install GNU Radio". So then we would have that, "PyBOMBS", and "From Source"  (or do you like "Source Code" better?).

Reply:
"Install" sounds good. Sub-topics also work nicely there. "From Source" makes sense. "PyBOMBS" makes sense too. Maybe "Prebuilt Binaries" for the other ... some we are distributing (Windows) ... yes? Some are for package managers (definitely). We might include links for others that are also pre-built. How does that sound to you?

Reply:
Rather than complicating the page here, I recommend putting that info in the Wiki: https://wiki.gnuradio.org/index.php/InstallingGR#From_Binaries
let's keep it simple here. My theme is "document in ONE place". :)
I will make the updates tonight or in the morning and do another push.

Reply:
I like that theme. Thanks for working on this! I look forward to the next iteration of this PR!